docs/reference/rollup/apis/put-job.asciidoc

```diff
@@ -1,3 +1,25 @@
+ifdef::permanently-unreleased-branch[]
+
+[role="xpack"]
+[testenv="basic"]
+[[rollup-put-job]]
+=== Create legacy {rollup-jobs} API
+[subs="attributes"]
+++++
+<titleabbrev>Create legacy {rollup-jobs}</titleabbrev>
+++++
+
+// tag::legacy-rollup-admon[]
+WARNING: This documentation is about legacy rollups. Legacy rollups are
+deprecated and will be replaced by new rollup functionality introduced in {es}
```

Comment:
I plan to add an xref to the updated overview page once that's ready. I've added a note to #65515.